my/src/data.ts

```diff
@@ -529,13 +530,13 @@ export const applicationQuestions: any = [
     requirements: {link: true},
   },
   {
-    label: "What's your devpost?",
+    label: "What's your Devpost?",
     fieldType: 'text',
     model: ['profiles', 'devpost'],
     requirements: {link: true},
   },
   {
-    label: 'Link us to any other profiles you have',
+    label: 'Is there anything else interesting you would want us to know? (e.g. Personal Website, Codepen, Dribble, Youtube, etc)',
     fieldType: 'text',
```

Comment:
Mind changing this to a text-area? That way ppl can hit enter and link multiple things

Reply:
Yeah we can do that